File: WarehouseManagementWeb.Application/Dto/Input/MeasureUnit/ChangeStatusMeasureUnitInput.cs
using WarehouseManagementWeb.Domain.Enums;

namespace WarehouseManagementWeb.Application.Dto.Input.MeasureUnit
{
    /// <summary>
    /// Класс входной модели обновления статуса единицы измерения.
    /// </summary>
    public class ChangeStatusMeasureUnitInput
    {
        /// <summary>
        /// Id единицы измерения.
        /// </summary>
        public int MeasureUnitId { get; set; }

        /// <summary>
        /// Статус единицы измерения.
        /// </summary>
        public DirectoryStatusEnum MeasureUnitStatusEnum { get; set; }
    }
}

File: warehouse-management-web-angular/src/app/directory/measure-unit/models/input/change-status-measure-unit-input.ts
import { DirectoryStatusEnum } from '../../../enums/directory-status-enum';

/**
 * Класс входной модели обновления статуса единицы измерения.
 */
export class ChangeStatusMeasureUnitInput {
  /**
   * Id единицы измерения.
   */
  measureUnitId: number = 0;

  /**
   * Статус единицы измерения.
   */
  measureUnitStatusEnum: DirectoryStatusEnum = 1;
}
